StigData/Processed/IISServer-8.5-1.7.xml

```diff
@@ -916,7 +916,7 @@ If any OS shell MIME types are configured, this is a finding.
       <IsNullOrEmpty>False</IsNullOrEmpty>
       <OrganizationValueRequired>False</OrganizationValueRequired>
       <OrganizationValueTestString />
-      <Path>%SystemDrive%\inetpub</Path>
+      <Path>%windir%\inetpub</Path>
```

Comment:
Looks like the %windir% made it back in this PR.

Reply:
fixed